test/functionalTest/harnessFunctions.sh

```diff
@@ -94,32 +94,45 @@ function dMsg()
 #
 function dbInit()
 {
-  role=$1  
-  db=$1
+  role=$1
+  
+  host="${CB_DATABASE_HOST}"
```

Comment:
Why did you need to change the way in which some harness functions are implemented? Or, in other words, which aspect in the current way of working make them incompatible with the docker stuff.